src/ecmascript_simd.js

```diff
@@ -120,6 +120,12 @@ function simdCreate(type) {
   return type.fn.apply(type.fn, lanes);
 }
 
+function simdCreateFromBuffer(type, buffer) {
+  var newValue = type.fn.apply(type.fn, lanes);
```

Comment:
'lanes' is not meaningful when this is called from simdFromBits - but it does create a new instance.